tests/AuthProvider/TokenTest.php

```diff
@@ -19,9 +19,9 @@ class TokenTest extends TestCase
 {
     public function testCreatingTokenAuthProvider()
     {
-        $options = $this->getOptions();
+        $options                     = $this->getOptions();
```

Comment:
Spaces are here also

Reply:
I guess it's a matter of preference but since it's your repo you decide 😃 
I have disabled alignment now.

Reply:
Thanks :+1: 
I also used such spaces some time ago.
But I found it didn't follow PSR standarts and didn't add readability in some cases like:
```php
$a                             = 1;
$bVeryLoooongNameOfTheVariable = 2;
$c                             = 3;
$d                             = 4;
```